.gitignore

```diff
@@ -1,10 +1,19 @@
-#  Local .terraform directories
+# Local .terraform directories
 **/.terraform/*
-**/.terraform.lock.hcl
 
 # .tfstate files
 *.tfstate
 *.tfstate.*
 
-# .tfvars files
-*.tfvars
+# Crash log files
+crash.log
+
+# Ignore override files as they are usually used to override resources locally and so
+# are not checked in
+override.tf
+override.tf.json
+*_override.tf
+*_override.tf.json
+
+# Lock file
+.terraform.lock.hcl
```

Comment:
> You should include this file in your version control repository so that you can discuss potential changes to your external dependencies via code review

Source: https://www.terraform.io/language/files/dependency-lock#lock-file-location

Reply:
Yes, we can track `.terraform.lock.hcl` in provision repo or inherited module, my idea is ignore tracking examples/*/.terraform.lock.hcl.
Or you think is it worthy to track it?

Reply:
Right... If this is only meant for examples, we can ignore it.